roles/openshift_master/templates/master.yaml.v1.j2

```diff
@@ -8,6 +8,12 @@ assetConfig:
   logoutURL: ""
   masterPublicURL: {{ openshift.master.public_api_url }}
   publicURL: {{ openshift.master.public_console_url }}/
+{% if openshift.master.loggingPublicURL %}
```

Comment:
This will use the value of loggingPublicURL set by openshift_facts, I do not see where that value is being set from this diff though.

Most likely you will also want to update roles/openshift_master/tasks.main.yml to include setting this value in the `openshift_facts` task as well.

I would also suggest using underscores rather than camelCase for the ansible variable names (logging_public_url instead of loggingPublicURL).